app/pages/home-common/featured-project.jsx

```diff
@@ -16,13 +16,13 @@ const FeaturedProject = (() =>
     <h1 className="secondary-kicker">Featured Project</h1>
     <div className="home-featured-images">
       <img
-        alt="Steelpan Vibrations"
-        src="/assets/featured-projects/featured-project-20170829-steelpan_vibrations.jpg"
+        alt="Planetarium Response Network & Rescue Global: Caribbean Storms 2017"
```

Comment:
"Planetarium" --> "Planetary"

Such an easy slip of the fingers when you work at Adler :)